scripts/deps_build.ts

```diff
@@ -147,11 +147,11 @@ async function build_dependency(lib_name: string, ...cmake_args: string[]) {
         '-DBUILD_SHARED_LIBS=OFF',
         '-DBUILD_TESTING=OFF',
         '-DCMAKE_INSTALL_LIBDIR=' + lib_dir,
-        '-DCMAKE_BUILD_TYPE=Release',
+        '-DCMAKE_BUILD_TYPE=Debug',
         cmake_args.join(' '),
         lib_source_dir,
     ].join(' ');
-    const build_cmd = ['cmake', '--build', './', '--config', 'release', '--target', 'install'].join(' ');
+    const build_cmd = ['cmake', '--build', './', '--config', 'debug', '--target', 'install'].join(' ');
```

Comment:
Probably want this toggleable base on some build param?
Or did you just not mean to check this in

Reply:
I'll make it a param or something. This is temp.